lib/cld/version.rb

```diff
@@ -1,3 +1,3 @@
 module CLD
-  VERSION = "0.7.0"
+  VERSION = "0.7.1"
```

Comment:
don't bump versions in PRs

Reply:
Sorry, totally missed that. I only bumped it so I could use my fork with a different version.